### ISSUE TITLE
Problem with character encoding

### ISSUE THREAD

**Issue by t3chn0m4g3** [vorband/ewsposter#20]
```
 => Starting Cowrie Modul.
 => Send Limit is set to : 500. Adapting to limit!
Traceback (most recent call last):
  File "/opt/ewsposter/ews.py", line 2609, in <module>
    eval(i+'()')
  File "<string>", line 1, in <module>
  File "/opt/ewsposter/ews.py", line 951, in cowrie
    "username"    : str(key[1]),
UnicodeEncodeError: 'ascii' codec can't encode character u'\xf6' in position 11: ordinal not in range(128)
```

I filtered for `username` which should be sufficient to follow up on the behavior and sent you a PM.

A fix for the encoding problem would be great, an error handling in cases like these would be even better since ewsposter crashes and will not send data for the rest of the day.

**Comment by vorband** [comment]
should be fixed in 34e5acb

**Comment by t3chn0m4g3** [comment]
thanks ... and rolling out :)
